examples/water-model/water-model.py

```diff
@@ -330,320 +354,269 @@
 
-    # Collect simulation data
-    trajectory.append(atoms.copy())
-    potential_energy[i_step] = atoms.get_potential_energy()
-    kinetic_energy[i_step] = atoms.get_kinetic_energy()
-    total_energy[i_step] = atoms.get_total_energy()
-    temperature[i_step] = atoms.get_temperature()
-    pressure[i_step] = -np.diagonal(atoms.get_stress(voigt=False)).mean()
 
 # %%
-#
-# Visualize the trajectory using `chemiscope <https://chemiscope.org>`_. For better
-# visualization, we include the unit cell.
+mol_data = get_bonds_angles(system.positions)    
+mol_data[-1]
 
-chemiscope.show(
-    trajectory,
-    mode="structure",
-    settings=chemiscope.quick_settings(
-        trajectory=True, structure_settings={"unitCell": True}
-    ),
-)
+m_sys = get_msites(system, 0.8, 0.85)
+p3m_calculator = torchpme.metatensor.P3MCalculator(
+                potential=torchpme.CoulombPotential(p3m_smearing),
+                **p3m_parameters
+            )
 
 # %%
-# Analyze the Results
-# -------------------
-#
-# Time Evolution of Physical Properties
-# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-#
-# We examine the time evolution of physical properties. For better comparison, an
-# arbitrary constant is subtracted from the potential energy.
-
-fig, ax = plt.subplots(3, figsize=(8, 5), sharex=True)
-
-time = 0.5 * np.arange(n_steps)
-
-ax[0].plot(time, potential_energy - potential_energy[-1], label="Potential Energy")
-ax[0].plot(time, kinetic_energy, label="Kinetic Energy")
-ax[0].plot(time, total_energy - potential_energy[-1], label="Total Energy")
-ax[0].legend(ncol=3)
-ax[0].set_ylabel("Energy [eV]")
-
-ax[1].plot(time, temperature, label="Temperature")
-ax[1].axhline(300, color="black", linestyle="--", label="Target Temperature")
-ax[1].legend(ncol=2)
-ax[1].set_ylabel("Temperature [K]")
-
-ax[2].plot(time, pressure)
-ax[2].axhline(0, color="black", linestyle="--")
-ax[2].set_ylabel("Pressure [eV Å$^{-3}$]")
-
-ax[-1].set_xlabel("Time [fs]")
-
-fig.align_labels()
-plt.show()
+p3m_calculator.forward(m_sys)
 
 # %%
-#
-# We find that temperature remains around the target of 300 K, and the pressure
-# stabilizes around a positive value.
-#
-# Radial Distribution Function
-# ^^^^^^^^^^^^^^^^^^^^^^^^^^^
-#
-# We compute and plot the partial radial distribution functions (RDF) between all pairs
-# in the system, using the last 20 steps to ensure at least some equilibration from the
-# initial configuration.
-
-OO_rdf_l = []
-OH_rdf_l = []
-HH_rdf_l = []
-
-for atoms in trajectory[-20:]:
-    OO_rdf_step, rdf_dists = ase.geometry.rdf.get_rdf(
-        atoms, rmax=6.0, nbins=200, elements=(8, 8)
-    )
-    OH_rdf_step, _ = ase.geometry.rdf.get_rdf(
-        atoms, rmax=6.0, nbins=200, elements=(8, 1)
-    )
-    HH_rdf_step, _ = ase.geometry.rdf.get_rdf(
-        atoms, rmax=6.0, nbins=200, elements=(1, 1)
-    )
-
-    OO_rdf_l.append(OO_rdf_step)
-    OH_rdf_l.append(OH_rdf_step)
-    HH_rdf_l.append(HH_rdf_step)
+class QTIP4PfModel(torch.nn.Module):
```

Comment:
@I liked the idea of having a general class for three and four point water models.

Also if this should be the one and only QTIP4P model there should be not parameters for the user to set. They are fixed by the definition of the model.

Reply:
I see your point here. I'll rename the class to TIP4P here and call the model qTIP4P/f. Re the 3-points, one can easily implement a 3-point by setting gamma to 0.